packages/dds/tree/src/feature-libraries/editable-tree/editableTree.ts

```diff
@@ -682,6 +718,21 @@ export class NodeProxyTarget extends ProxyTarget<Anchor> {
 
 		return { parent: proxifiedField, index };
 	}
+
+	public on<K extends keyof EditableTreeEvents>(
+		eventName: K,
+		listener: EditableTreeEvents[K],
+	): () => void {
+		assert(eventName === "changed", "unexpected eventName");
+		const unsubscribeFromValueChange = this.anchorNode.on("valueChange", () => listener());
+		const unsubscribeFromChildrenChange = this.anchorNode.on("childrenChange", () =>
+			listener(),
+		);
+		return () => {
+			unsubscribeFromValueChange();
+			unsubscribeFromChildrenChange();
+		};
+	}
```

Comment:
Why does the proxy deliver different events than the anchor?

Reply:
The customer facing API on editableTree is going to want lots of different events, event bubbling stuff, event deferral logic etc. I'm not attempting to present a good or final API there, just enough for the people working in that area to get started.